src/util/formatter.ts

```diff
@@ -111,6 +113,33 @@ function sumMultiBalanceOf(balances, results, chain = '', provider = '') {
   }
 }
 
+function swapTokenAddresses(
+  tokens: { [key: string]: any },
+  rule: {
+    [key: string]: {
+      address: string;
+      decimals: number;
+    };
+  },
+) {
+  Object.keys(tokens).forEach((address) => {
+    if (rule[address]) {
+      if (tokens[rule[address].address]) {
```

Comment:
I think you can just add` || 0` to the next line and you won't need "else" part in this if:
` tokens[rule[address].address] = BigNumber(tokens[rule[address].address] || 0)`